content/blog/bootstrapping-urbit-from-ethereum.md

```diff
@@ -378,10 +378,9 @@ price in ether of a planet, and the number of planets available.
 
 ## Okay, so
 
-That was quite a lot of information!  We've created a fora thread
-[here](https://urbit.org/fora/posts/~2017.9.20..22.56.04..47ce~) for discussion,
-and would be glad to hear your feedback.  GitHub issues on the
-[urbit/constitution repo](https://github.com/urbit/constitution) are also more
+That was quite a lot of information! There's also a blog post about the constitution
+[here](https://urbit.org/blog/interim-constitution/), and GitHub issues on the
+[urbit/constitution repo](https://github.com/urbit/constitution) are more
```

Comment:
This link is also dead — links to a private repo. Where does our actual constitution information live now?

Reply:
Perhaps we could just make the repo public? @galenwp what do you think?

Reply:
@galenwp Ping.

Reply:
@Fang- will know which repo is correct. I think https://github.com/urbit/azimuth ? We called it the constitution for a while before changing it.

Reply:
@Fang- ping

Reply:
Azimuth is indeed the obviously correct repo here.